textproc/libxmlxx5/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           meson 1.0
+
+
+# requires C++11 as of version 2.39.1
+compiler.cxx_standard 2017
+
+name                libxmlxx5
+set gname           libxml++
+version             5.0.1
+set branch          [join [lrange [split ${version} .] 0 1] .]
+categories          textproc
+license             LGPL
+maintainers         openmaintainer
+description         libxml++ is a C++ interface for working with XML files
+long_description    libxml++ is a C++ interface for working with XML files, using \
+                    libxml (gnome-xml) to parse and write the actual XML files. It \
+                    has a simple but complete API.
+homepage            https://libxmlplusplus.github.io/libxmlplusplus/
+platforms           darwin
+distname            ${gname}-${version}
+master_sites        gnome:sources/${gname}/${branch}/
+
+patchfiles-append   patch-meson-build.diff
+
+use_xz              yes
+
+checksums           rmd160  2bd00b4b9b2ff059954ac834a824d4ca2e2831a9 \
+                    sha256  15c38307a964fa6199f4da6683a599eb7e63cc89198545b36349b87cf9aa0098 \
+                    size    779760
+
+depends_lib-append  port:libxml2 \
+					port:pkgconfig
+                    
+
+livecheck.type      gnome
+livecheck.name      ${gname}
+
+variant mantainer description {Enable mantainer mode.} {
+    configure.args-append       -Dmaintainer-mode=true
+    depends_build-append        port:mm-common
+}
+
+variant docs description {Build documentation.} {
+    configure.args-append       -Dbuild-documentation=true
+    depends_build-append        port:perl5 \
+                                port:doxygen \
+                                port:graphviz \
```

Comment:
this should likely be `path:bin/dot:graphviz` so that also `graphviz-devel` can satisfy the dependency